README.md

```diff
@@ -411,65 +440,585 @@ Four actions are available for you to hook onto through the tooltip/popover life
+  });
+
+});
+```
+
+#### Test helper option: `selector`
+
+The selector of the tooltip or popover you are testing.
+
+If more than one tooltip or popover is found in the DOM when you run an assertion, you will be asked to specify this.
+
+| Type    | String |
+|---------|---------|
+| Default | `'.ember-tooltip, .ember-popover'` |
+
+Usage example:
+
+```js
+import { assertTooltipRendered } from 'appname/tests/helpers/ember-tooltips';
+
+test('Example test', function(assert) {
+
+  this.render(hbs`
+    {{tooltip-on-element class='test-tooltip'}}
+    {{tooltip-on-element}}
+  `);
+
+  assertTooltipRendered(assert, {
+    selector: '.test-tooltip',
+  });
+});
+```
+
+#### Test helper option: `targetSelector`
+
+The selector of the tooltip or popover target you are testing. See [Targets](#targets) for an explanation on what a 'target' is.
+
+If more than one tooltip or popover target is found in the DOM when you run an assertion, you will be asked to specify this.
+
+| Type    | String |
+|---------|---------|
+| Default | `'.ember-tooltip-or-popover-target'` |
+
+Usage example:
+
+```js
+import { assertTooltipSpacing } from 'appname/tests/helpers/ember-tooltips';
+
+test('Example test', function(assert) {
+
+  this.render(hbs`
+    <div class="test-target">
+      {{tooltip-on-element}}
+    </div>
+
+    <div>
+      {{tooltip-on-element}}
+    </div>
+  `);
+
+  assertTooltipSpacing(assert, {
+    targetSelector: '.test-target',
+  });
+});
+```
+
+#### Test helper option: `side`
+
+The value for the tooltip or popover's [`side` option](#side) that you are asserting.
+
+| Type    | String |
+|---------|---------|
+| Default | `null |
+
+For example, if you specify for the tooltip or popover be shown on the right of the target using `side='right'`, you will pass `side: 'right'` in assertions that test side. Here is the code for this example:
+
+```js
+import { assertTooltipSide } from 'appname/tests/helpers/ember-tooltips';
+
+test('Example test', function(assert) {
+
+  this.render(hbs`
+    {{tooltip-on-element side='right'}}
+  `);
+
+  assertTooltipSide(assert, {
+    side: 'right',
+  });
+});
+```
+
+#### Test helper option: `spacing`
+
+The value for the tooltip or popover's [`spacing` option](#spacing) that you are asserting. Specify as a number of pixels expected (without a `px` unit).
+
+| Type    | Number |
+|---------|---------|
+| Default | `null` |
+
+For example, if you specify for the tooltip or popover be shown on the right of the target using `side='right'`, you will pass `side: 'right'` in assertions that test side. Here is the code for this example:
+
+```js
+import { assertTooltipSide } from 'appname/tests/helpers/ember-tooltips';
+
+test('Example test', function(assert) {
+
+  this.render(hbs`
+    {{tooltip-on-element spacing='35'}}
+  `);
+
+  assertTooltipSpacing(assert, {
+    spacing: 35,
+  });
+});
+```
+
+#### Test helper option: `event`
+
+The name of the event that you would like to trigger on an element.
```

Comment:
This may need to be updated depending on the outcome of this discussion.

https://github.com/sir-dunxalot/ember-tooltips/pull/160/files#diff-9693234231881212e47f7813b6502a69R118